README.md

```diff
@@ -185,30 +189,105 @@ guarantees.
+
+### Exporter
+
+| Component                                                   | Artifact ID                                | Version                                                     |
+|-------------------------------------------------------------|--------------------------------------------|-------------------------------------------------------------|
+| [OTLP gRPC Exporter](./exporters/otlp/all)                  | `opentelemetry-exporter-otlp`              | <!--VERSION_STABLE-->1.14.0<!--/VERSION_STABLE-->           |
+| [OTLP gRPC Trace Exporter](./exporters/otlp/trace)          | `opentelemetry-exporter-otlp-trace`        | <!--VERSION_STABLE-->1.14.0<!--/VERSION_STABLE-->           |
+| [OTLP gRPC Metrics Exporter](./exporters/otlp/metrics)      | `opentelemetry-exporter-otlp-metrics`      | <!--VERSION_STABLE-->1.14.0<!--/VERSION_STABLE-->           |
+| [OTLP gRPC Logs Exporter](./exporters/otlp/logs)            | `opentelemetry-exporter-otlp-logs`         | <!--VERSION_UNSTABLE-->1.14.0-alpha<!--/VERSION_UNSTABLE--> |
+| [OTLP HTTP Trace Exporter](./exporters/otlp-http/trace)     | `opentelemetry-exporter-otlp-http-trace`   | <!--VERSION_STABLE-->1.14.0<!--/VERSION_STABLE-->           |
+| [OTLP HTTP Metrics Exporter](./exporters/otlp-http/metrics) | `opentelemetry-exporter-otlp-http-metrics` | <!--VERSION_STABLE-->1.14.0<!--/VERSION_STABLE-->           |
+| [OTLP HTTP Logs Exporter](./exporters/otlp-http/logs)       | `opentelemetry-exporter-otlp-http-logs`    | <!--VERSION_UNSTABLE-->1.14.0-alpha<!--/VERSION_UNSTABLE--> |
+| [OTLP Common](./exporters/otlp/common)                      | `opentelemetry-exporter-otlp-common`       | <!--VERSION_STABLE-->1.14.0<!--/VERSION_STABLE-->           |
+| [Jaeger gRPC Exporter](./exporters/jaeger)                  | `opentelemetry-exporter-jaeger`            | <!--VERSION_STABLE-->1.14.0<!--/VERSION_STABLE-->           |
+| [Jaeger Thrift Exporter](./exporters/jaeger-thrift)         | `opentelemetry-exporter-jaeger-thift`      | <!--VERSION_STABLE-->1.14.0<!--/VERSION_STABLE-->           |
+| [Jaeger Proto](./exporters/jaeger-proto)                    | `opentelemetry-exporter-jaeger-proto`      | <!--VERSION_STABLE-->1.14.0<!--/VERSION_STABLE-->           |
+| [Logging Exporter](./exporters/logging)                     | `opentelemetry-exporter-logging`           | <!--VERSION_STABLE-->1.14.0<!--/VERSION_STABLE-->           |
+| [OTLP Logging Exporter](./exporters/logging-otlp)           | `opentelemetry-exporter-logging-otlp`      | <!--VERSION_STABLE-->1.14.0<!--/VERSION_STABLE-->           |
+| [Zipkin Exporter](./exporters/zipkin)                       | `opentelemetry-exporter-zipkin`            | <!--VERSION_STABLE-->1.14.0<!--/VERSION_STABLE-->           |
+| [Prometheus Exporter](./exporters/prometheus)               | `opentelemetry-exporter-prometheus`        | <!--VERSION_UNSTABLE-->1.14.0-alpha<!--/VERSION_UNSTABLE--> |
+
+### Shim
```

Comment:
Maybe add a one-sentence description of what we mean by "shim".

Reply:
I think with the added description column these have become self explanatory.